src/seedu/addressbook/AddressBook.java

```diff
@@ -733,17 +733,6 @@ private static void addPersonToAddressBook(HashMap<PersonProperty, String> perso
     }
 
     /**
-     * Deletes a person from the address book, target is identified by it's absolute index in the full list.
-     * Saves changes to storage file.
-     *
-     * @param index absolute index of person to delete (index within {@link #ALL_PERSONS})
-     */
-    private static void deletePersonFromAddressBook(int index) {
-        ALL_PERSONS.remove(index);
-        savePersonsToFile(getAllPersonsInAddressBook(), storageFilePath);
-    }
-
-    /**
      * Deletes the specified person from the addressbook if it is inside. Saves any changes to storage file.
```

Comment:
take note of your commit messages; they should be "imperative"
e.g: **Remove** unused method as against **Removed** unused method